Demo/DemoTests/SignalProducer+MoyaSpec.swift

```diff
@@ -183,7 +183,7 @@ class SignalProducerMoyaSpec: QuickSpec {
                 }
                 
                 expect(receivedError).toNot(beNil())
-                expect(receivedError?.domain).to(equal("\(NSCocoaErrorDomain)"))
+//                expect(receivedError?.domain).to(equal("\(NSCocoaErrorDomain)"))
```

Comment:
Hmm, good question. I've not compared enum cases in Nimble matchers before. I looks like [a question others have had](https://github.com/Quick/Nimble/issues/212). If you want, take a stab at one of the solutions @jeffh suggests there – I can lend a hand to get these working, too.